src/dev/konnector.md

```diff
@@ -39,179 +56,168 @@ yarn # or npm install
-    name: 'toto',
-    height: 1.8
-  },
-  {
-    name: 'titi',
-    height: 1.7
+```
+{
+  "name": "konnector name",
+  "type": "node",
+  "slug": "konnectorslug",
+  "description": "description",
+  "source": "git://github.com/cozy/cozy-konnector-thename.git",
+  "permissions": {
+    "accounts": {
+      "description": "Required to get the account's data",
+      "type": "io.cozy.accounts",
+      "verbs": ["GET"]
+    }
   }
-]
-
-return filterData(data, 'io.cozy.height', {
-  keys: ['name']
-}).then(filteredData => addData(filteredData, 'io.cozy.height'))
+}
 ```
 
-** addData **
+The cozy-konnector-template already has a manifest which you can customize.
 
-This function save the data into the cozy
+You may add some permissions for your own doctype. [Here](https://cozy.github.io/cozy-stack/konnectors.html) is the detailed list of fields for a
+connector manifest file.
 
-Parameters:
+You can also get more information on permissions in the official [cozy-stack documentation](https://github.com/cozy/cozy-stack/blob/master/docs/permissions.md)
 
- - `entries`: an array of objects corresponding to the data you want to save to the cozy
- - `doctype`: the doctype where you want to save data (ex: 'io.cozy.bills')
+### konnector-dev-config.json
 
-** saveFiles **
+If you want to put data from your connector to a real cozy, your must define where to find this
+cozy, and this must be a cozy for which you have the credentials.
 
-The goal of this function is to save the given files in the given folder via the Cozy API.
+Here comes the `COZY_URL` in konnector-dev-config.json which defines just that.
 
- - `entries`: an array of objects where the function will look for fileurl and filename attributes
- - `fields`: the object passed to the fetch function which contains the folderPath attributes which
- - `options`: options to influence the behavior of the saveFiles function. See the [detailed
-   documentation](https://github.com/cozy/cozy-konnector-libs/blob/master/docs/api.md#savefiles)
-   for more information about it
+Then you just have to run:
 
-** updateOrCreate` **
-
-The goal of this function is create or update the given entries according to if they already exist
-in the cozy or not
-
-Parameters:
-
- * - `entries` is an array of objects with any attributes :
- *
- * - `doctype` (string) is the cozy doctype where the entries should be saved
- *
- * - `filters` (array) is the list of attributes in each entry should be used to check if an entry
- *   is already saved in the cozy
+```sh
+yarn dev
+```
 
-### The manifest
+And for the first run, the cli will open a tab in your browser asking you to give permissions to the
+connector and the connector will save data directly in your cozy. This will validate that your
+manifest has the needed permissions.
 
-Each connector is described by a Manifest. This is a JSON file named `manifest.konnector` at the root of your code folder. It should include the following information:
+this is the *dev* mode
```

Comment:
This is the *development* mode.

Reply:
This was on purpose, since the standalone mode is also a development mode, I am not very satisfied about these name, but I don't think this is the good time to change it.